lib/utils.js

```diff
@@ -53,27 +53,6 @@ exports.isString = function(obj) {
   return typeof obj === 'string';
 };
 
-/**
- * Watch the given `files` for changes
- * and invoke `fn(file)` on modification.
- *
- * @private
- * @param {Array} files
- * @param {Function} fn
- */
-exports.watch = function(files, fn) {
-  var options = {interval: 100};
-  var debug = require('debug')('mocha:watch');
-  files.forEach(function(file) {
-    debug('file %s', file);
-    fs.watchFile(file, options, function(curr, prev) {
-      if (prev.mtime < curr.mtime) {
-        fn(file);
-      }
-    });
-  });
-};
-
```

Comment:
`utils.files` is not referenced anymore, so can be deleted as well.

Reply:
Awesome! Fixed in https://github.com/mochajs/mocha/pull/3980/commits/55df5546a53a5f9cbf23db54423ef3a6bfec5e04